apps/wakunode2/wakunode2.nim

```diff
@@ -92,20 +94,19 @@ when isMainModule:
     error "4/7 Mounting protocols failed", error=res5.error
     quit(QuitFailure)
 
+proc startNode*() =
```

Comment:
Exaclty, that is a simple split. However that might not be needed, and I will likely revert it back, when I wrap the `waku_node.nim` library instead of the wakunode2 app.